### PR TITLE
Cache elementScreenshotSupport per browser

### LINKED ISSUE #1261: ScreenshotComparator#supportsElementScreenshots shouldn't be static
`supportsElementScreenshots` depends on the browser used for test.
Currently it will be set depending on the first browser used to take an element screenshot and that setting will be used for other browsers.
Ideally it should be cached per capability. Alternatively cache should be removed.

### PR DESCRIPTION
**Pull request by alvarezguille**
Fixes #1261

### CODE REVIEW

File: vaadin-testbench-core/src/main/java/com/vaadin/testbench/commands/ScreenshotComparator.java [hunk 1]
```diff
@@ -26,7 +28,7 @@
 
 public class ScreenshotComparator {
 
-    private static Boolean supportsElementScreenshots = null;
+    private static Set<String> browsersWithoutElementScreenshot = new HashSet<>();
```
Comment (Artur-) [inline vaadin-testbench-core/src/main/java/com/vaadin/testbench/commands/ScreenshotComparator.java:31]: Can this be accessed by multiple threads at the same time? Looks like there could be concurrency issues

Comment (alvarezguille) [reply vaadin-testbench-core/src/main/java/com/vaadin/testbench/commands/ScreenshotComparator.java]: Done

**Comment by Artur-** [comment]
How is this tested?

**Comment by alvarezguille** [comment]
Tests are already failing for IE and FF in the build for all browsers
image
image
>org.openqa.selenium.UnsupportedCommandException: GET /session/XXXXXXXX-XXXX-XXXX-XXXX-XXXX72480538/element/0/screenshot
Build info: version: '3.4.0', revision: 'unknown', time: 'unknown'
System info: host: 'WIN-R1JBAO1CNPS', ip: '10.100.16.13', os.name: 'Windows Server 2008 R2', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_91'
Driver info: driver.version: unknown
Command duration or timeout: 0 milliseconds
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
	at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
	at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
	at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:285)
	at org.openqa.selenium.remote.RemoteWebElement.getScreenshotAs(RemoteWebElement.java:392)
	at com.vaadin.testbench.TestBenchElement.getScreenshotAs(TestBenchElement.java:392)
	at com.vaadin.testbench.commands.ScreenshotComparator.getScreenshot(ScreenshotComparator.java:97)
	at com.vaadin.testbench.commands.ScreenshotComparator.compareScreen(ScreenshotComparator.java:42)
	at com.vaadin.testbench.TestBenchElement.compareScreen(TestBenchElement.java:459)
	at com.vaadin.tests.elements.ElementScreenCompareIT.elementCompareScreen(ElementScreenCompareIT.java:43)

**Pull request merged**